aerosense_tools/preprocess.py

```diff
@@ -153,8 +153,7 @@ def plot(self, sensor_types_metadata, sensor_names=None, plot_start_offset=dt.ti
         :return: plotly.graph_objs.Figure: a line graph of the sensor data against time
         """
 
-        if not plot_max_time:
-            plot_max_time = self.duration-plot_start_offset
+        plot_max_time = plot_max_time or self.duration-plot_start_offset
```

Comment:
Do you have the pre-commit hooks set up? They should have prevented no spacing around operators like this

Reply:
No, I have no hooks set up for this project.

Reply:
Could you run `pre-commit install && pre-commit install -t commit-msg`? It will automate a lot of basic code review